Gemfile

```diff
@@ -162,4 +162,6 @@ group :development do
   gem 'pry'
   gem 'pry-debugger', :platform => [:ruby_19, :ruby_21]
   gem 'pry-byebug', :platform => :ruby_20
+  gem 'awesome_print', require:'ap'
+  gem 'meta_request'
```

Comment:
This gem with RailsPanel chrome extension is useful to debug the application using chrome.
More information at https://github.com/dejan/rails_panel